examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>I would rename it to something like "Advanced: Define your own architecture"</p><p> </p><p>We can write that we explain it at the example of DLRM</p><p>We do not want to use build in the context of training - building recommender systems includes the 4 stages.</p><p> </p><p>Same for learning objective - Define a model architecture from scratch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I see, thanks for the clarification! I updated all the texts using <code>build</code>  and renamed the notebook as well!</p>